lxc/console.go

```diff
@@ -320,6 +330,12 @@ func (c *cmdConsole) vga(d lxd.InstanceServer, name string) error {
 		return err
 	}
 
+	// Cancel the operation if the context is canceled.
+	go func() {
+		<-ctx.Done()
+		_ = op.Cancel()
```

Comment:
in a normal scenario, what cancels the operation when its done?

Can we make it a normal part of the cleanup, rather than a separate go routine?